### PR TITLE
Add binary examples for Textcat task in `spacy-llm`

### PR DESCRIPTION
**Pull request by rmitsch**
## Description

Add binary examples for Textcat task in `spacy-llm`.

### Types of change

Expand docs.

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [ ] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: website/docs/api/large-language-models.mdx [hunk 1]
```diff
@@ -752,6 +784,22 @@ supports `.yml`, `.yaml`, `.json` and `.jsonl`.
...
+    "text": "You won the lottery! Wire a fee of 200$ to be able to withdraw your winnings.",
+    "answer": "Spam"
+  },
+  {
+    "text": "Your order #123456789 has arrived",
+    "answer": "NotSpam"
+  }
+]
+```
+
```
[the remark this answers is not in the document]
Comment (rmitsch) [reply website/docs/api/large-language-models.mdx:803]: I misremembered how the textcat task works for binary classification - there is an explicit code/prompt path that relies on "POS"/"NEG" responses and uses only one class. Fixed in https://github.com/explosion/spaCy/pull/13051/commits/3cb57e18e107d137c6e8ded81dd587178308293e.

File: website/docs/api/large-language-models.mdx [hunk 2]
```diff
@@ -673,6 +673,22 @@ prompt.
...
```
[the remark this answers is not in the document]
Comment (rmitsch) [reply website/docs/api/large-language-models.mdx]: Done in https://github.com/explosion/spaCy/pull/13051/commits/64c40d16f06534d36ca84806441879195c9bc697.

File: website/docs/api/large-language-models.mdx [hunk 3]
```diff
@@ -752,6 +752,24 @@ supports `.yml`, `.yaml`, `.json` and `.jsonl`.
 path = "textcat_examples.json"
 ```
 
+If you want to perform few-shot learning with a binary classifier (i. e. a text
+either should or should not be assigned to a given class), you can provide
+positive and negative examples with the POS/NEG label. An example for spam
```
Comment (svlandeg) [inline website/docs/api/large-language-models.mdx:757]: Should we clarify that the "actual" label, the one referring to "POS", is the one defined in the "labels" argument of the task?

Comment (rmitsch) [reply website/docs/api/large-language-models.mdx]: Done in https://github.com/explosion/spaCy/pull/13051/commits/f6d9e5c4dfe6c70ecfc868906949e1a0d85d15b3.

**Comment by svlandeg** [comment]
Final nitpick :-)

This is added documentation unrelated to an upcoming `spacy-llm` release, so in fact it should target spaCy's `master` and upon merging, can be cherry-picked straight through to `spacy.io` so it comes live ASAP.

**Review by svlandeg — APPROVED** [review]

**Pull request merged**